src/openvic-simulation/politics/Issue.cpp

```diff
@@ -101,9 +123,22 @@ bool IssueManager::_load_reform_group(
 	return ret;
 }
 
-bool IssueManager::_load_reform(size_t& ordinal, std::string_view identifier, ReformGroup const* group, ast::NodeCPtr node) {
-	// TODO: conditions to allow, policy modifiers, policy rule changes
-	return add_reform(identifier, group, ordinal);
+bool IssueManager::_load_reform(
+	ModifierManager const& modifier_manager, RuleManager const& rule_manager, size_t ordinal, std::string_view identifier,
+	ReformGroup const* group, ast::NodeCPtr node
+) {
+	// TODO: conditions to allow, policy rule changes
+	ModifierValue values;
+	RuleSet rules;
+	Reform::tech_cost_t technology_cost;
```

Comment:
```suggestion
	Reform::tech_cost_t technology_cost = 0;
```
Default initialise, otherwise on MSVC it will contain random data.